liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -643,10 +651,11 @@ private List<String> getServletNames(WebModule webModule) throws DeploymentExcep
          List<String> servletNames = new ArrayList<String>();
          getServletNames(webModule.archive, servletNames);
 
+         boolean isServletRunner = "servlet".equalsIgnoreCase(containerConfiguration.getTestProtocol()); 
```

Comment:
I think it would be cleaner to add a `isServletProtocol()` and `isRestProtocol()` method to the configuration classes and use those everywhere that you are doing a hardcoded "servlet" or "rest" `equalsIgnoreCase` method call.

Reply:
I updated the PR with an isServletProtocol() method.  Today there are just the two so I didn't see a need for adding the isRestProtocol() method at this time unless you think it would make sense to do so.